### ISSUE TITLE
Parametrization missing

### ISSUE THREAD

**Issue by suculent** [suculent/apprepo#5]
1. Retrive existing metadata from AppRepo's metadata.json (requires given APPCODE, but theoretically could list all apps in numbered list and let user choose, or create new one [good old days[)

2. Use given IPA file path, or result from fastlane (or find last one in current directory as this will not fail like on testflight). 

3. Compile as much metadata (must: version, build, title) as possible to build the json file with respective metadata (notify, or metadata.json details, if the binary upload succeeds) and upload to AppRepo as well (trigger notification if required).

4. Post the rescan procedure on AppRepo side. Should happen before step 3 with asynchronous end/start trigger.

**Comment by suculent** [comment]
Rescan remaining.

**Comment by suculent** [comment]
Duplicate of https://github.com/suculent/apprepo/issues/9
